setup.py

```diff
@@ -50,6 +50,7 @@
         'Flask-CORS>=2.1.0',
     ],
     'docs': [
+        'dicttoxml>=1.7.4'
```

Comment:
needed to build the doc because `autodoc` will try to load `examples/app.py`, and fail if `dicttoxml` isn't installed...
This dependency also appear in the `requirement.txt` of the example app.